src/DomainReader.ts

```diff
@@ -166,7 +166,7 @@ export class DomainReader {
   }
 
   protected reviveDates(key: string, value: string | number | Date): string | number | Date {
-    if (key === "minDate" || key === "maxDate") {
+    if ((key === "minDate" || key === "maxDate") && value !== null) {
```

Comment:
As I understand when minDate and maxDate are left empty field is not constrained. Does null values of max and min will allow user to enter arbitrary date?

Reply:
yeah, I tested passing null values for min and max date and the front end allowed me to enter an arbitrary dates.

Reply:
only thing is that when min and max values are null , they should not be passed to new Date(), as it will return 1970-01-01 in that case